spec/inputs/syslog_spec.rb

```diff
@@ -1,5 +1,5 @@
 # encoding: utf-8
-require "logstash/devutils/rspec/spec_helper"
+require_relative "../spec_helper"
```

Comment:
I particulary find more ugly to add the code in the specs, that you did that before it does not mean i find it nice :+1: ... the usage of spec_helpers for test and move code there is a nice things, the specs should only have specs, not fix/configuration like this ... even if it's just 1 line.